GetIntoTeachingApi/Models/BaseModel.cs

```diff
@@ -118,6 +118,14 @@ protected void NotifyPropertyChanged([CallerMemberName] string propertyName = ""
             }
         }
 
+        protected void DeleteLink(ICrmService crm, OrganizationServiceContext context, BaseModel childToRemove, string childToRemovePropertyName)

```

Comment:
Can we rename to `modelToRemove` and `propertyNameContainingModelToRemove`? `child` implies a parent/child relationship which isn't always going to be the case